timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -209,39 +218,48 @@ def fit(
             used to compute the validation scores. Note that only the last ``prediction_length`` time steps of each
             time series are used for computing the validation score.
 
-            If ``None``, AutoGluon will split :attr:`train_data` into training and tuning subsets using
+            If None, AutoGluon will split :attr:`train_data` into training and tuning subsets using
             ``self.validation_splitter``. If ``tuning_data`` is provided, ``self.validation_splitter`` will be ignored.
             See the description of ``validation_splitter`` in the docstring for
             :class:`~autogluon.timeseries.TimeSeriesPredictor` for more details.
+
+            # TODO: Mention that this is a good default that users should not override?
+            # TODO: Link to advanced tutorial explaining how to do more fancy validation / how validation works?
         time_limit : int, default = None
-            Approximately how long :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will run for (wall-clock
-            time in seconds). If not specified, :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will
-            run until all models have completed training.
-        presets : str, default = None
+            Approximately how long :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will run (wall-clock time in
+            seconds). If not specified, :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will run until all models
+            have completed training.
+        presets : str, default = "medium_quality"
             Optional preset configurations for various arguments in
             :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit`.
 
+            # TODO: Better explanation of how to choose presets
             Can significantly impact predictive accuracy, memory footprint, inference latency of trained models,
             and various other properties of the returned predictor. It is recommended to specify presets and avoid
             specifying most other :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` arguments or model
             hyperparameters prior to becoming familiar with AutoGluon. For example, set ``presets="best_quality"``
             to get a high-accuracy predictor, or set ``presets="low_quality"`` to get a toy predictor that
             trains quickly but lacks accuracy.
+            # TODO: This looks like a legacy sentence, remove it? Are we referring to the `hyperparameters` kwarg here?
             Any user-specified arguments in :meth:`~autogluon.timeseries.TimeSeriesPredictor.fit` will
             override the values used by presets.
 
+            # TODO: Should we even mention low_quality_hpo?
```

Comment:
I think we could even remove the preset low quality HPO :)

Reply:
and I'm happy to just do this as a breaking change as I doubt if any users depend on it